### DISCUSSION TITLE
Favourites breadcrumbs (browser QA)

### DISCUSSION THREAD

**Discussion by eemila** [po8klasie/po8klasie#80] [UX-QA]
Breadcrumbs are misleading. 
Once the user chooses one of his fav schools we can only get back (using breadcrumbs) to Search and Home - not Favourites, where he came from.

**Comment by karmek-k** [comment]
I suggest adding a request parameter that indicates where the user has come from:
`https://test.warsawlo.pl/school/886?from=favourites`
and modify the page's state based on that.

It might be also possible to inspect the router's history (as one can do with React Router, but idk about Reach Router).

**Comment by micorix** [comment]
How about adding some property to location state and handling it "internally" :slightly_smiling_face:  ? 

Additional URL params here would be quite a mess IMO. Besides, user (A) can copy that link and send it to someone (B). Considering this case, user B would see the breadcrumbs, although they shouldn't.

**Comment by micorix** [comment]
Also this issue is just for QA. Let's move technical discussion to another issue.
